babelsubs/loader.py

```diff
@@ -65,18 +65,24 @@ def add_region(self, xml_id, style_id, **attrib):
         """
         self.regions.append((xml_id, style_id, attrib))
 
-    def _empty_ttml(self, language_code, title, description):
+    def _empty_ttml(self, language_code, title, description, frame_rate=None,
+                    frame_rate_multiplier=None):
         if not self.styles:
             raise ValueError("no styles added")
         if not self.regions:
             raise ValueError("no regions added")
-
-        tt = etree.Element(TTML + 'tt', attrib={
-            XML + 'lang': language_code,
-        }, nsmap = {
+        attrib = {}
+        if language_code:
+            attrib[XML + 'lang'] = language_code
+        if frame_rate:
+            attrib[TTP + 'frameRate'] = frame_rate
+            if frame_rate_multiplier:
+                attrib[TTP + 'frameRateMultiplier'] = frame_rate_multiplier
+        tt = etree.Element(TTML + 'tt', attrib=attrib, nsmap={
```

Comment:
not sure of the best way to handle this, but this would be referenced as lxml.etree.Element (we still use lxml for building elements, since the alternative would be to re-implement the C bindings lxml is a wrapper for, or redesign a significant amount of functionality) post-xml fixes, since we use defusedxml.ElementTree as etree elsewhere